Misc/NEWS.d/next/C API/2020-05-26-23-06-59.bpo-40778.7cumE-.rst

```diff
@@ -0,0 +1,2 @@
+Update :c:macro:`Py_SIZE` and :c:macro:`Py_SIZE` to use const object
+casting. Patch by Dong-hee Na.
```

Comment:
I don't think that it deserves to be documented. It doesn't change the C API, no?